### ISSUE TITLE
Why is external concentration of ca2 1mM not 2mM? 

### ISSUE THREAD

**Issue by pgleeson** [OpenSourceBrain/SolinasEtAl-GolgiCell#8]
To reproduce, type ca2o in the original NEURON code. cao is 2mM.

This code: https://github.com/OpenSourceBrain/SolinasEtAl-GolgiCell/blob/master/NEURON/Golgi_template.hoc#L86 may have been meant to set ca2o to 2mM, but setup of mechanisms in Nrn is tricky.

@theiera is it ok to change ca2o to 2mM?

@RokasSt this may have been the cause of some of the discrepancy with CaLVA in NML2...

**Comment by RokasSt** [comment]
I am using 2mM in NeuroML2,  as NEURON is showing.  See  here
https://github.com/RokasSt/SolinasEtAl-GolgiCell/blob/master/NeuroML2/TestSoma.cell.nml

ca2o is set to 2mM.

I will check whether the NeuroML2 results of running the LVA channel will change with 1 mM.

@pgleeson 
@theiera

**Comment by theiera** [comment]
This code: https://github.com/OpenSourceBrain/SolinasEtAl-GolgiCell/blob/master/NEURON/Golgi_template.hoc#L86  is
cai0_ca_ion = 50e-6
and it is correct.

I guess you meant: https://github.com/OpenSourceBrain/SolinasEtAl-GolgiCell/blob/master/NEURON/Golgi_template.hoc#L92
ca2o = cao

and cao should be 2 mM by default.

**Comment by RokasSt** [comment]
@pgleeson  so we should keep using 2 mM

**Comment by pgleeson** [comment]
@theiera, yes that's the line that is supposed to set ca2o, but it doesn't (my link was to the general block where concentrations are set). A correct way to set the initial value of ca2 (which is an unrecognised ion; cao defaults to 2, all other unrecognised ions default to 1) is:

```
ca2o0_ca2_ion = 2
```

This will make sure ca2o is always 2.

Note changing ca2o to 2mM will change the behaviour of all NEURON examples slightly (spikes change ~0.5%) The options now are: 

a) leave the error in place, put 1mM in the NML2 for ca2o & 2mM for cao
b) fix it in the NEURON, update the [tests](https://github.com/OpenSourceBrain/SolinasEtAl-GolgiCell/tree/master/NEURON/test) and visually check the graphs again
